packages/browser/src/integrations/remoteconfig.ts

```diff
@@ -0,0 +1,101 @@
+import { defineIntegration, getClient, remoteConfig } from '@sentry/core';
+import type { IntegrationFn, RemoteConfigStorage, RemoteOverrideableConfig } from '@sentry/types';
+import { logger } from '@sentry/utils';
+import { DEBUG_BUILD } from '../debug-build';
+import { WINDOW } from '../helpers';
+
+const INTEGRATION_NAME = 'RemoteConfig';
+
+/**
+ * Remote Configuration fetches configuration from a remote server.
+ */
+const _remoteConfigIntegration = (() => {
+  let _inst: ReturnType<typeof remoteConfig> | undefined;
+  return {
+    name: INTEGRATION_NAME,
+    setupOnce() {
+      const client = getClient();
```

Comment:
use `setup(client)` instead here:

```suggestion
    setup(client) {
```